modules/system/ServiceProvider.php

```diff
@@ -97,6 +97,13 @@ public function boot()
             }
         }
 
+        /*
+         * Set a default samesite config value for invalid values
+         */
+        if (!in_array(strtolower(Config::get('session.same_site')), ['lax', 'strict', 'none'])) {
+            Config::set('session.same_site', 'Lax');
+        }
```

Comment:
@ayumi-cloud one final question, why do we need this in the first place? Wouldn't all browsers just default to lax anyways? Can you think of any browsers that would accept and treat null as it's own separate value with distinct behaviour? 

Just don't want to get into the situation where someone with a specific use case that only uses an older browser would be relying on that value remaining null and then all of a sudden we prevent that from happening.

Reply:
@LukeTowers 

> one final question, why do we need this in the first place? Wouldn't all browsers just default to lax anyways? Can you think of any browsers that would accept and treat null as it's own separate value with distinct behaviour?

IE11 uses a different method for SameSite than Chrome, Edge, Firefox, Opera and Safari. So this bit of code work help IE11 as a fallback to prevent a null operation in their browser.

Reply:
I believe the IE11 pr was here: https://github.com/MicrosoftEdge/Status/issues/616